### PR TITLE
Revolt. She Said. Revolt Again.

### PR DESCRIPTION
**Pull request by samozzy**
Programme info, in order as given in the programme.

### CODE REVIEW

File: _shows/16_17/revolt_she_said.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+cast:
+- role: Dinah
+  name: Daniella Finch
+- role: Grandma
+  name: Lois Baglin
+- role: Agnes
+  name: Chloe Richardson
+- role: Multi-Role
+  name: Harry Pavlou
+
```
Comment (BenWoodford76) [inline _shows/16_17/revolt_she_said.md]: ``

File: _shows/16_17/revolt_she_said.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+cast:
+- role: Dinah
+  name: Daniella Finch
+- role: Grandma
+  name: Lois Baglin
+- role: Agnes
+  name: Chloe Richardson
+- role: Multi-Role
+  name: Harry Pavlou
```
Comment (BenWoodford76) [inline _shows/16_17/revolt_she_said.md]: Shouldn't all the cast have a  'multi-role' after them, Dinah, Grandma and Agnes are only their characters in Act 3

Comment (samozzy) [inline _shows/16_17/revolt_she_said.md]: That's a question for the show team - this list is from the programme.

**Comment by samozzy** [comment]
Is it not worth keeping this branch live given its more general purpose name? Is that not better for history?

**Pull request merged**